src/app/config.ts

```diff
@@ -1 +1,2 @@
-export const BACKEND_V1_URL = "http://localhost:3000/api/v1";
+//export const BACKEND_V1_URL = "http://localhost:3000/api/v1";
+export const BACKEND_V1_URL = "http://ucab-profile-backend.playoffside.online/api/v1";
```

Comment:
Although this will be solved by merging [improvement/use-token-in-middleware](https://github.com/luizzzito/ER-frontend/tree/improvement/use-token-in-middleware) branch with this one (since in that branch it is already solved)
```suggestion
export const BACKEND_V1_URL = "https://ucab-profile-backend.playoffside.online/api/v1";
```

Reply:
SOLUCIONADO 👍